src/Symfony/Component/ErrorRenderer/Resources/views/logs.html.php

```diff
@@ -16,7 +16,10 @@
         } elseif ($log['priority'] >= 300) {
             $status = 'warning';
         } else {
-            $severity = $log['context']['exception']['severity'] ?? false;
+            $severity = false;
+            if (($exception = $log['context']['exception'] ?? null) && $exception instanceof \ErrorException) {
```

Comment:
`if (($exception = $log['context']['exception'] ?? null) instanceof \ErrorException) {` should be enough, right?

Reply:
that worked as well, fixed